### PR TITLE
Network version12 additional fixes

### PR DESCRIPTION
**Pull request by lachlan-git**
I am aware there are a few issues just putting thise here and asking for assistance

### CODE REVIEW

**Comment by lachlan-git** [comment]
New node ID, is one more then the previous max node in county:
image

**Comment by lachlan-git** [comment]
FT 4 1 lane each way

image

**Comment by lachlan-git** [comment]
where output node goes too:

image

File: project_cards/year_2015_antioch_bridge_extension.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+project: year 2015 antioch bridge extension
+tags:
+- year 2015
```
Comment (i-am-sijia) [inline project_cards/year_2015_antioch_bridge_extension.yml:3]: add a `highway review` tag

File: project_cards/year_2015_antioch_bridge_extension.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    county: Contra Costa
+    drive_access: 1
+    model_link_id: 2517015
+    rail_only: 0
+    walk_access: 1
+    lanes: 2
+    assignable: 1
+    ft: 4
+    roadway: primary
+
```
Comment (i-am-sijia) [inline project_cards/year_2015_antioch_bridge_extension.yml:32]:
```suggestion
```

File: project_cards/year_2015_antioch_bridge_extension.yml (hunk 1 above)
Comment (lachlan-git) [reply project_cards/year_2015_antioch_bridge_extension.yml:3]: Was waiting on notebook to run to commit,.

committed now, notebook is running

File: project_cards/year_2015_antioch_bridge_extension.yml (hunk 2 above)
Comment (lachlan-git) [reply project_cards/year_2015_antioch_bridge_extension.yml]: removed white space

**Pull request merged**